mr_2022_desktop_existing.toml

```diff
@@ -0,0 +1,30 @@
+## EXPERIMENT SPECIFIC SETTINGS
+# [experiment]
+
+## NEW METRICS (uncomment below)
+# [metrics]
+
+
+## MODIFICATIONS TO DEFAULT DATA SOURCES TO ACCOUNT FOR EARLY ENROLLMENT
+# these cover all metrics in defaults/firefox_desktop.toml except unenrollment
+# which we do not override, to include unenrollment counts from prior to release
+[data_sources]
+
+[data_sources.clients_daily]
+from_expression = """(
+    SELECT 
+        p.*
+    FROM mozdata.telemetry.clients_daily p
+    WHERE mozfun.norm.extract_version(p.app_display_version, 'major') >= 106
+)"""
+
+[data_sources.search_clients_engines_sources_daily]
+from_expression = """(
+    SELECT 
+        p.*
+    FROM mozdata.search.search_clients_engines_sources_daily p
+    WHERE mozfun.norm.extract_version(p.app_display_version, 'major') >= 106
+)"""
```

Comment:
@irrationalagent I believe that this should exclude any data from all metrics from before the release goes live, can you confirm this?

Reply:
Since we are only enrolling clients on the release channel, that should avoid us getting any data from profiles that were also active on beta/nightly 106 prior to the release. So yes, I think this should work.